src/servicebus/azext_servicebus/tests/recordings/Testing.py

```diff
@@ -0,0 +1,9 @@
+import isodate
+from isodate.isoerror import ISO8601Error
+from knack.util import CLIError
+
+try:
+    isodate.parse_duration('PT11qM')
+except ISO8601Error as s:
+    raise CLIError(s)
+
```

Comment:
Looks like this file was used for debugging. Delete?
If it is supposed be be kept, it shouldn't be in the `recordings` directory and shouldn't start with a capital letter.

Reply:
deleted.